### PR TITLE
Add Loom

### PR DESCRIPTION
**Pull request by ysm-dev**
https://www.loom.com/

- https://vercel.com/
- https://www.hashicorp.com/

Not sure if "Digital Asset Management" is correct, please check.

### CODE REVIEW

File: src/technologies/l.json [hunk 1]
```diff
@@ -1472,6 +1472,22 @@
     "scriptSrc": "loja2\\.com\\.br",
     "website": "https://www.loja2.com.br"
   },
+  "Loom": {
+    "cats": [
+      95
```
Comment (nurbek91) [inline src/technologies/l.json:1477]: Hey mate 103 Livestreaming more suitable for this technology
"description":  "Loom is a video recording and screen sharing solution.",

Comment (ysm-dev) [reply src/technologies/l.json]: It's not a kind of "Live"streaming, but if there is no related category.. 🤔

[33b1b22](https://github.com/wappalyzer/wappalyzer/pull/7345/commits/33b1b222608be4d7291cf373122d25188e014261)

Comment (nurbek91) [reply src/technologies/l.json]: The difference between Loom and Zoom is that Loom is a video messaging tool and Zoom is a video conferencing tool.

Comment (nurbek91) [reply src/technologies/l.json]: live chat maybe

Comment (ysm-dev) [reply src/technologies/l.json]: [53513e9](https://github.com/wappalyzer/wappalyzer/pull/7345/commits/53513e9c00dfb06311fa5797bcb18194da9dbd53)
Added more detection method

Comment (ysm-dev) [reply src/technologies/l.json]: example: https://docs.cal.com/guides/creating-your-first-event-type

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**